primo-explore/custom/01NWU_INST-HSLV/css/custom1.css

```diff
@@ -1807,3 +1807,21 @@ prm-atoz-search-bar {
 .facet-title-row .text-number-space {
   font-weight: normal;
 }
+
+#qrCodeScanner md-icon svg[id^='qrCode_'] {
+  display:none;
+}
+
+prm-search-bookmark-filter button#qrCodeScanner {
```

Comment:
Are `prm-search-bookmark-filter` and others like it, custom Primo elements? They don't just use classes :open_mouth: